app/uk/gov/hmrc/plasticpackagingtax/registration/forms/ConfirmAddress.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.forms
+
+import play.api.data.Form
+import play.api.data.Forms.{mapping, text}
+import play.api.libs.json.{Format, JsBoolean, Reads, Writes}
+
+case class ConfirmAddress(useRegisteredAddress: Boolean)
+
+object ConfirmAddress extends CommonFormValidators {
+
+  implicit val format: Format[ConfirmAddress] =
+    Format[ConfirmAddress](Reads.StringReads.map(ConfirmAddress.apply),
+                           Writes[ConfirmAddress](obj => JsBoolean(obj.useRegisteredAddress))
+    )
+
+  lazy val emptyError = "primaryContactDetails.jobTitle.empty.error"
+  val yes             = "yes"
+  val no              = "no"
+
+  def form(): Form[ConfirmAddress] =
+    Form(
+      mapping(
+        "useRegisteredAddress" -> text()
+          .verifying(emptyError, contains(Seq("yes", "no")))
+      )(ConfirmAddress.apply)(ConfirmAddress.unapply)
+    )
+
+  def apply(value: String): ConfirmAddress =
+    if (value == "yes")
```

Comment:
can probably use val's here instead of repeating strings

Reply:
Done